mscore/scoreview.cpp

```diff
@@ -3613,12 +3613,6 @@ void ScoreView::contextPopup(QContextMenuEvent* ev)
                   objectPopup(gp, e);
                   }
             }
-      else {
```

Comment:
I'm not sure it's a good idea, anyway, to pop up commands to append and prepend (like the title frame) when you could be pages and pages away from either end of the score.